pkgs/development/python-modules/flask-simpleldap/default.nix

```
@@ -0,0 +1,38 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, flask
+, python-ldap
+}:
+
+buildPythonPackage rec {
+  pname = "flask-simpleldap";
+  version = "1.4.0";
+  format = "setuptools";
```

```suggestion
  pyproject = true;
```

Package does have `pyproject.toml` in source repository. So it's better to use it.
During build, I also got a bunch of warnings, suggesting the use of PEP-517 installer.

Also don't forget to add:
```
  build-system = [
    setuptools
  ];
```

---

v2 came out after I created the PR 🙈

I've fixed the code review comments (:

---

Even if the project only has `setup.py`, I guess it's still better to use `pyproject`. Since python installer gives out warnings of setuptools deprecation. And in future will fail to build on newer versions of python.

While [pypa/build](https://github.com/pypa/build) which is used when `pyproject = true;` has a default values for `pyproject.toml` if it's not present and will handle legacy package correctly.